package.json

```diff
@@ -24,7 +24,8 @@
     "prettier-check": "yarn prettier --check '+(e|web)/**/*.{ts,tsx,js,jsx,mts}'",
     "prettier-write": "yarn prettier --write --log-level silent '+(e|web)/**/*.{ts,tsx,js,jsx,mts}'",
     "process-icons": "node web/packages/design/src/Icon/script/script.js & yarn prettier --loglevel silent --write 'web/packages/design/src/Icon/Icons/*.tsx'",
-    "nop": "exit 0"
+    "nop": "exit 0",
+    "postinstall": "patch-package"
```

Comment:
The pnpm PR will have to account for this as well, right? https://pnpm.io/cli/patch

Reply:
Correct. From what I understand, we will simply remove `patch-package` as `pnpm` supports patching out of the box.

Reply:
It looks like pnpm needs some entries in package.json for patches to work though. https://youtu.be/0GjLqRGRbcY?t=137